src/components/c-breadcrumb.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div :class="b()">
     <ul :class="b('list')">
-      <li v-for="(item, index) in getItems"
+      <li v-for="(item, index) in itemsInternal"
```

Comment:
suggestion: I would recommend to directly use `breadcrumbStore.items` and drop the getter.